clang/include/clang/AST/TemplateBase.h

```diff
@@ -168,6 +168,8 @@ class TemplateArgument {
     LLVM_PREFERRED_TYPE(bool)
     unsigned IsDefaulted : 1;
     uintptr_t V;
+    LLVM_PREFERRED_TYPE(bool)
+    unsigned IsCanonicalExpr : 1;
```

Comment:
Can we put this ABOVE the `V` here?  `uintptr_t` on some platforms has a 64 bit alignment, so there is potentially a ton of padding above `V`.

Reply:
Yeah sure. Though it's not going to make a difference in practice because other members of the union are much larger.